Maple2.File.Ingest/Mapper/ItemMapper.cs

```diff
@@ -2,19 +2,29 @@
 using Maple2.File.IO;
 using Maple2.File.Parser;
 using Maple2.File.Parser.Xml.Item;
+using Maple2.File.Parser.Xml.Table;
 using Maple2.Model.Enum;
 using Maple2.Model.Metadata;
+using Maple2.Tools.Extensions;
 
 namespace Maple2.File.Ingest.Mapper;
 
 public class ItemMapper : TypeMapper<ItemMetadata> {
     private readonly ItemParser parser;
+    private readonly TableParser tableParser;
 
     public ItemMapper(M2dReader xmlReader) {
         parser = new ItemParser(xmlReader);
+        tableParser = new TableParser(xmlReader);
     }
 
     protected override IEnumerable<ItemMetadata> Map() {
+        IEnumerable<(int Id, ItemExtraction Extraction)> itemExtractionData = tableParser.ParseItemExtraction();
```

Comment:
nit: just inline this, the temporary variable is not useful here.

But actually I think you can do even better:
```cs
Dictionary<int, ItemExtraction> itemExtractionDictionary = tableParser.ParseItemExtraction()
    .ToDictionary(entry => entry.Id, entry => entry.Extraction);
````
or even:
```cs
Dictionary<int, int> itemExtractionTryCount = tableParser.ParseItemExtraction()
    .ToDictionary(entry => entry.Id, entry => entry.Extraction.TryCount);
````
since that's seems to be all you need here.